### DISCUSSION TITLE
/articles/sharepoint-embedded-2024-roadshow/

### DISCUSSION THREAD

**Discussion by giscus** [Voitanos/voitanos.io-comments#309] [Comments]
Join a SharePoint Embedded exclusive event for developers, architects, & decision makers who want to create document-centric apps with SharePoint Embedded.
https://www.andrewconnell.com/articles/sharepoint-embedded-2024-roadshow/

**Comment by Spucelik** [comment]
Thanks for Posting!!

**Reply by andrewconnell** [reply]
Absolutely Steve!
